soh/soh/SohGui/SohGui.cpp

```diff
@@ -120,17 +120,15 @@ namespace SohGui {
         mSohMenu = std::make_shared<SohMenu>(CVAR_WINDOW("Menu"), "Port Menu");
         gui->SetMenu(mSohMenu);
 
-        mStatsWindow = gui->GetGuiWindow("Stats");
-        if (mStatsWindow == nullptr) {
-            SPDLOG_ERROR("Could not find stats window");
-        }
-
         mConsoleWindow = std::make_shared<SohConsoleWindow>(CVAR_WINDOW("SohConsole"), "Console##SoH", ImVec2(820, 630));
         gui->AddGuiWindow(mConsoleWindow);
 
         mGfxDebuggerWindow = std::make_shared<SohGfxDebuggerWindow>(CVAR_WINDOW("SohGfxDebugger"), "GfxDebugger##SoH", ImVec2(820, 630));
         gui->AddGuiWindow(mGfxDebuggerWindow);
 
+        mStatsWindow = std::make_shared<SohStatsWindow>(CVAR_WINDOW("SohStats"), "SohStats", ImVec2(400, 100));
```

Comment:
To avoid having "Soh" displayed in the window title (when popped out). Same change needs to be applied in the menu listing.

```suggestion
        mStatsWindow = std::make_shared<SohStatsWindow>(CVAR_WINDOW("SohStats"), "Stats##Soh", ImVec2(400, 100));
```